var/spack/repos/builtin/packages/wrf/package.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+import glob
+import tempfile
+from os.path import dirname
+import multiprocessing as mp
+
+
+class Wrf(AutotoolsPackage):
```

Comment:
It looks like this is a "fake" Autotools package, as in they don't actually use Autotools. I would extend `Package` and then add:
```python
phases = ['configure', 'build', 'install']
```
See https://spack.readthedocs.io/en/latest/build_systems/custompackage.html for more info.